lib/mobius/validations/action_validations.ex

```diff
@@ -87,6 +93,7 @@ defmodule Mobius.Validations.ActionValidations do
   def get_validator({:integer, opts}), do: get_integer_range_validator(opts)
   def get_validator(:string), do: string_validator()
   def get_validator({:string, opts}), do: get_string_length_validator(opts)
+  def get_validator({module, function}), do: fn val -> apply(module, function, [val]) end
```

Comment:
```suggestion
  def get_validator(validator), do: validator
```
I just checked, we _could_ have this take a function directly, but the custom validators would have to be defined outside of the the actions module in order to be able to capture them in the endpoint definition. Not sure which one is better :thinking:

Reply:
Ah because it's a macro the module itself isn't defined yet when the macro is executed, that's unfortunate

I prefer having the option to put the custom validator in the module itself, makes it a little bit less spaghetti I guess